packages/components/src/local/hex-grid/index.tsx

```diff
@@ -59,6 +64,17 @@ export const HexGrid: React.FC<PropTypes> = ({ gridCells }: PropTypes) => {
         centres[hex.name] = centreWorld
       })
 
+      // create a polygon for each hex, add it to the parent
+      const plannedRoutePoly: L.LatLng[] = []
```

Comment:
This looks like it could be a map and filter.

Reply:
I know it's a performance hog.  

Once this PR is out of the way, I think I need to do an implementation change.  We're passing these lists of hexes round as string cell-names.

That means in too many places we have to do: `gc.find(cell => cell.name == listCellName)`

I think that instead, i should be passing around the actual lists of Cells, not cell names.